zookeeper-server/src/main/java/org/apache/zookeeper/ZooKeeperMain.java

```diff
@@ -249,6 +261,17 @@ protected void connectToZK(String newHost) throws InterruptedException, IOExcept
             System.setProperty(ZKClientConfig.SECURE_CLIENT, "true");
             System.out.println("Secure connection is enabled");
         }
+        if (cl.getOption("waitforconnection") != null) {
```

Comment:
Pls add documentation in zookeeperCLI.md

Reply:
added documentation for waitforconnection